src/components/app/Home.js

```diff
@@ -78,6 +102,44 @@ class UploadButton extends React.PureComponent<UploadButtonProps> {
           />
           Select a profile to open
         </label>
+        <label
+          id="LoadFromUrlButton"
+          className="homeSectionButton"
+          onClick={this._loadFromUrlPressed}
+        >
+          Load profile from an URL
+        </label>
+        {this.state.isLoadFromUrlPressed ? (
+          <UploadFromUrl {...this.props} />
+        ) : null}
+      </div>
+    );
+  }
+}
+
+class UploadFromUrl extends React.PureComponent<UploadFromUrlProps> {
+  _input: HTMLInputElement | null;
+
+  _takeInputRef = input => {
+    this._input = input;
+  };
+
+  _upload = () => {
+    if (this._input) {
+      this.props.triggerLoadingFromUrl(this._input.value);
+    }
+  };
+  render() {
+    return (
+      <div className="homeSectionUploadFromUrl">
```

Comment:
please make this `position: absolute; display: flex`: the first will avoid that the UI "jumps" when it appears, the second will make the `input` and `button` takes the whole space, hopefully.